src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectPreferencesDialog.java

```diff
@@ -153,16 +157,74 @@ public void onError(ServerError error)
       
    }
    
-   private void emitRenvConsoleActions(RProjectRenvOptions options)
+   private void emitRenvConsoleActions(RProjectRenvOptions newOptions)
    {
-      if (options.useRenv == renvOptions_.useRenv)
+      EventBus events = pEventBus_.get();
+      
+      // renv is off and staying off -- nothing to do
+      if (!newOptions.useRenv && !renvOptions_.useRenv)
+      {
```

Comment:
Perhaps return in this empty block? It looks like there's no code that executes if this block is hit.